src/main/java/zmaster587/advancedRocketry/dimension/DimensionProperties.java

```diff
@@ -812,15 +813,13 @@ public void tick() {
 				satallites.remove(satallite.getId());
 			}
 		}
-
-		this.prevOrbitalTheta = this.orbitTheta;
-		this.orbitTheta += (201-orbitalDist)*0.000005d;
+        updateOrbit();
 	}
 
 
 	public void updateOrbit() {
-		this.prevOrbitalTheta = this.orbitTheta;
-		this.orbitTheta = (AdvancedRocketry.proxy.getWorldTimeUniversal(getId())*(201-orbitalDist)*0.000002d) % (2*Math.PI);
+		StellarBody parentStar = new StellarBody();
+		this.orbitTheta = AstronomicalBodyHelper.getOrbitalTheta(orbitalDist, parentStar.getSize());
```

Comment:
We should probably be using the parent star's size, not a brand new star with default updates every tick.  Consider `getStar()`?